www/lynx/Portfile

```diff
@@ -3,9 +3,9 @@
 PortSystem          1.0
 
 name                lynx
-set milestone       2.8.9
-version             ${milestone}rel.1
-revision            2
+set milestone       2.9.0
+version             ${milestone}
+revision            0
 categories          www
 platforms           darwin
```

Comment:
this is the default since MacPorts v2.7 and can/should thus be removed

Reply:
You appear to be referring to the `milestone` keyword, which I find only mentioned in a commit comment from 2008.

Reply:
To be fair, he didn't change the `platforms` line himself.